mmdet3d/datasets/pipelines/__init__.py

```diff
@@ -1,6 +1,6 @@
 from mmdet.datasets.pipelines import Compose
 from .dbsampler import DataBaseSampler
-from .formating import Collect3D, DefaultFormatBundle, DefaultFormatBundle3D
+from .formating import Collect3D, DefaultFormatBundle3D
```

Comment:
This may influence nuImages models.

Reply:
This will not as nuImages models should use a similar module as that in mmdet.